### DISCUSSION TITLE
How long time are you taking to build a plot?

### DISCUSSION THREAD

**Discussion by WMark77** [Chia-Network/chia-blockchain#6111] [General]
Hello,

I'm testing Chia on a machine with a 600GB NVMe. It's taking about 2 days per k-32 plot. Does it slow? 

Regards,

**Comment by tiny0little** [comment]
https://github.com/Chia-Network/chia-blockchain/wiki/Reference-Plotting-Hardware

**Reply by WMark77** [reply]
Thank you.

**Comment by chiabug** [comment]
Two days with NVME seems to be excessive.  What OS and what processor?   On low end hardware, using regular drives, a 14 - 28 hour plot time is common.  My experience is that a NVME in the same hardware only decreases the plot time by about 10 percent.

**Reply by WMark77** [reply]
It's an emulated arm64 Linux system. I think Chia performance depends on CPUs much more than we guessing.

**Comment by aleckzts** [comment]
I am using a 500GB HDD WD Black as temporary space and it takes 16h per plot.

**Reply by WMark77** [reply]
Thanks for replying.

**Reply by aleckzts** [reply]
I was making 2 plots per disk, now that I have just 1 plot per disk it takes 10-12h to complete.. I think its acceptable

**Reply by hibari0614** [reply]
> I was making 2 plots per disk, now that I have just 1 plot per disk it takes 10-12h to complete.. I think its acceptable

@aleckzts 
Would you mind share the specification of HDD (model number) ?

**Reply by aleckzts** [reply]
For temp I am using WD Black 500Gb WD5000LPSX and WD Purple as final destination.

**Reply by hibari0614** [reply]
Thanks your kindly reply.

**Comment by hibari0614** [comment]
Using disk RAID will decrease the plotting time efficiently. I am using a spinning disk RAID with 2x 500GB WD disks (7200 RPM), it takes about 11h for per k=32 plot.

Below is the rig that I uses for plotting.
    HP Pavilion DV6 7017-TX:
        CPU : Intel i7-3610QM (4c8t, 6M cacha)
        RAM : DDR3 - 1600 16 GB (8GB x 2)
    Temporary space:
        RAID 0: 2x 500GB WD disk (WD5003ABYZ, 7200 RPM), a external storage that link my laptop with USB 3.0

**Comment by rtm125** [comment]
I plot with 2x 1TB NVME PCI-E 4.0 drives, I plot 6-10 files concurrently and produce about 16-20 a day [plotting picture on imgur](https://imgur.com/a/zMieMBe) The files are then moved to one of 3 different storages drives after plotting is completed. I started about 7 days ago now and have 6TB full. The first two days I didn't have the second NVME or the USB drives and was only making about 4-6 plots per day.

I also have my Ethereum miner plotting direct to a USB drive and that thing, if it is lucky gets like 1 plot a day. Horrid piece of junk but has no problems mining Ethereum 24/7 for the past 6 months.

Only been doing this for a week and all have this haunting feeling about this coin being a fresh pump and dump, but we'll see. I hope it isn't. Really wanted to get at least 100TB on the network.
